packages/core/src/reporters/ProgressKeeper.ts

```diff
@@ -32,32 +32,36 @@ abstract class ProgressKeeper implements Reporter {
     }
   }
 
+  protected getElapsedTime() {
+    const elapsedSeconds = this.timer.elapsedSeconds();
+
+    return this.formatTime(
+      // hours:
+      Math.floor(elapsedSeconds / 3600),
+      // minutes up to 2 decimal places:
+      Math.floor(((elapsedSeconds % 3600) * 100) / 60) / 100
+    );
+  }
+
   protected getEtc() {
     const totalSecondsLeft = Math.floor((this.timer.elapsedSeconds() / this.progress.tested) * (this.progress.total - this.progress.tested));
 
     if (isFinite(totalSecondsLeft) && totalSecondsLeft > 0) {
-      const hours = Math.floor(totalSecondsLeft / 3600);
-      const minutes = Math.floor((totalSecondsLeft / 60) % 60);
-      const seconds = Math.floor(totalSecondsLeft % 60);
-
-      return this.formatEtc(hours, minutes, seconds);
+      return this.formatTime(
+        // minutes up to 2 decimal places:
+        Math.floor(totalSecondsLeft / 3600),
+        // minutes up to 2 decimal places:
+        Math.floor(((totalSecondsLeft % 3600) * 100) / 60) / 100
+      );
     } else {
       return 'n/a';
     }
   }
 
-  private formatEtc(hours: number, minutes: number, seconds: number) {
-    let output;
-
-    if (hours > 0) {
-      output = `${hours}h, ${minutes}m, ${seconds}s`;
-    } else if (minutes > 0) {
-      output = `${minutes}m, ${seconds}s`;
-    } else {
-      output = `${seconds}s`;
-    }
-
-    return output;
+  private formatTime(hours: number, minutes: number) {
```

Comment:
I still think there is a place for seconds. I don't think showing minutes as `1.75` is helpful as people think in seconds, not parts of a minute. However, I do agree that showing seconds if time is > ~10 minutes doesn't make sense. 

So to summarize:

* If > 1h show hours and minutes
* else if > 10 minutes, show minutes (rounded to full)
* else show minutes and seconds

@brodybits do you agree?

Reply:
Yes for estimated remaining time, I am still not convinced to do this for elapsed time.

I would agree that we don't really want minutes with decimal places, already reverted the change and added "[WIP]" to the title until we sort this out.

I would like to continue this conversation in the normal comments on this PR.